include/igl/opengl/glfw/imgui/ImGuiMenu.cpp

```diff
@@ -185,7 +185,7 @@ IGL_INLINE void ImGuiMenu::draw_viewer_menu()
   // Workspace
   if (ImGui::CollapsingHeader("Workspace", ImGuiTreeNodeFlags_DefaultOpen))
   {
-    float w = ImGui::GetContentRegionAvailWidth();
+    float w = ImGui::GetContentRegionAvail().x;
```

Comment:
I have added the `IMGUI_DISABLE_OBSOLETE_FUNCTIONS` define macro when updating our imgui dependency, which required updating the code accordingly. I could make a separate PR for this but I don't think it's worth the effort (adding `IMGUI_DISABLE_OBSOLETE_FUNCTIONS` to our current CMake would create conflict, and otherwise the change is not needed).